Summary (required)

Elena-Hrechyk opened the pull request:
Перевірте будь ласка

Elena-Hrechyk committed:
Summary (required)

ceREAL133 commented on routes/api/contacts.js:
```diff
@@ -1,25 +1,85 @@
-})
+router.get("/", async (req, res, next) => {
+  try {
+    const result = await contacts.listContacts();
```
router should only invoke function or middleware, but not define it

ceREAL133 commented on routes/api/contacts.js:
```diff
@@ -1,25 +1,85 @@
+
+router.put("/:contactId", async (req, res, next) => {
+  try {
+    const { error } = schema.validate(req.body);
```
same with this and another routes

Elena-Hrechyk committed:
Summary (required)

Fix: created controllers and middlewares
routes/api/contacts.js only invoke  function and middlewares
